cmd/thor/flags.go

```diff
@@ -150,6 +150,10 @@ var (
 		Value: "localhost:2112",
 		Usage: "metrics service listening address",
 	}
+	enableDebug = cli.BoolFlag{
+		Name:  "enable-debug",
```

Comment:
Suggestions to fall in line with the other flags:
- `enable-api-debug`
- `api-enable-debug`